ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+
+cat > "${PATCH}" << EOF
+featureSet: TechPreviewNoUpgrade
+credentialsMode: Passthrough
+platform:
+  gcp:
+    computeSubnet: ${HOST_PROJECT_COMPUTE_SUBNET}
+    controlPlaneSubnet: ${HOST_PROJECT_CONTROL_SUBNET}
+    createFirewallRules: Disabled
+    network: ${HOST_PROJECT_NETWORK}
+    networkProjectID: ${HOST_PROJECT}
+    publicDNSZone:
+      id: installer-public-zone
+      project: ${HOST_PROJECT}
```

Comment:
I think we should remove this. We want to test the configuration where the DNS records are created in the service project, which will be the default. So this can just be deleted.

Last time we tested creating dns records in the service project, masters failed to pull ignition. Not sure why. Working on setting up a local environment where we can test this.